From 24ba1ce7214cc5573c82ba6670340b9ec579a2e8 Mon Sep 17 00:00:00 2001
From: jkreierh <jkreierh@90b65887-3827-0410-9a23-83215b262276>
Date: Thu, 30 Jun 2016 07:56:38 +0000
Subject: [PATCH 0689/1000] MANTIS 40719 Mantis 40719: CR 006: Fast force
 inject - Introduce new transaction "inject" with CMD_INJECT and
 IND_INJECT_DONE

git-svn-id: svn://localhost/SelfServiceCommon/trunk@1039 90b65887-3827-0410-9a23-83215b262276
---
 .../Massai/idl/Massai/ConveyorDev.idl         | 22 +++++++++++--------
 1 file changed, 13 insertions(+), 9 deletions(-)

diff --git a/SelfServiceCommon/Massai/idl/Massai/ConveyorDev.idl b/SelfServiceCommon/Massai/idl/Massai/ConveyorDev.idl
index fcec24da..e2152485 100644
--- a/SelfServiceCommon/Massai/idl/Massai/ConveyorDev.idl
+++ b/SelfServiceCommon/Massai/idl/Massai/ConveyorDev.idl
@@ -39,7 +39,7 @@
 **/
 module conveyordev
 {
-   const string interface_version = "1.11";
+   const string interface_version = "1.12";
 /*
 using DeviceComponents::indicationListener and DeviceComponents::Generic from mdci.idl
   RC indicationListener::indication(in any ind);
@@ -55,7 +55,7 @@ using DeviceComponents::indicationListener and DeviceComponents::Generic from md
     with cmd: DeviceComponents::command
       with cmd.type:   conveyordev::commandType
       with cmd.data:   conveyordev::bagInfo                 for CMD_DELIVERY_NOTE_SET, CMD_PIECE_RFID_WRITE
-      with cmd.data:   string                               for CMD_SESSION_BEGIN       
+      with cmd.data:   string                               for CMD_SESSION_BEGIN
       with cmd.data:   DeviceComponents::nullData           for the others
 */
 
@@ -395,12 +395,15 @@ using DeviceComponents::indicationListener and DeviceComponents::Generic from md
       CMD_PARKING_FORWARD,                   /**< transport a piece from parking belt to main belt  */
       CMD_PARKING_PROCESS,                   /**< read properties of the baggage on parking belt  */
       CMD_PARKING_RETURN,                    /**< transport a piece from parking belt to verification belt  */
-      CMD_SESSION_BEGIN,                     /**< start a session, 
-                                                  this command can be called parallel to other commands, 
-                                                  no response indication will be sent (RC_ACCEPTED is not allowed) */
-      CMD_SESSION_END                        /**< end of the current session,
-                                                  this command can be called parallel to other commands, 
-                                                  no response indication will be sent (RC_ACCEPTED is not allowed) */
+      CMD_SESSION_BEGIN,                     /**< start a session,
+                                                  this command can be called parallel to other commands,
+                                                  no response indication will be sent
+                                                  (RC_ACCEPTED is not allowed) */
+      CMD_SESSION_END,                       /**< end of the current session,
+                                                  this command can be called parallel to other commands,
+                                                  no response indication will be sent
+                                                  (RC_ACCEPTED is not allowed) */
+      CMD_INJECT                             /**< inject a bag with checks disabled. */
    };
 
    /**
@@ -446,7 +449,8 @@ using DeviceComponents::indicationListener and DeviceComponents::Generic from md
       IND_PIECE_IMAGE,                       /**< image(s) of the piece. */
       IND_PIECE_NOREAD,                      /**< No data read form any label. */
       IND_PIECE_DELIVERED,                   /**< Piece has left kiosk-controlled conveyors. */
-      IND_PIECE_DIMENSIONS                   /**< Piece-height, length, or width. */
+      IND_PIECE_DIMENSIONS,                  /**< Piece-height, length, or width. */
+      IND_INJECT_DONE                        /**< Finished CMD_INJECT */
    };
 
    /**
-- 
2.41.0.windows.1

